opm/simulators/flow/LogOutputHelper.cpp

```diff
@@ -99,132 +126,83 @@ cumulative(const std::size_t reportStepNum) const
 {
     this->beginCumulativeReport_();
 
-    std::vector<Scalar> tmp_values(WellCumDataType::numWCValues, 0.0);
-    std::vector<std::string> tmp_names(WellCumDataType::numWCNames, "");
+    using Ix = typename WellCumDataType::WCId;
 
     const auto& st = summaryState_;
-    for (const auto& gname : schedule_.groupNames()) {
-        auto gName = static_cast<std::string>(gname);
-        auto get = [&st, &gName](const std::string& vector)
-        {
-            const auto key = vector + ':' + gName;
-            return st.has(key) ? st.get(key) : 0.0;
-        };
-
-        tmp_names[0] = gname;
-
-        if (tmp_names[0] == "FIELD") {
-            tmp_values[2] = st.get("FOPT", 0.0); // WellCumDataType::OilProd
-            tmp_values[3] = st.get("FWPT", 0.0); // WellCumDataType::WaterProd
-            tmp_values[4] = st.get("FGPT", 0.0); // WellCumDataType::GasProd
-            tmp_values[5] = st.get("FVPT", 0.0); // WellCumDataType::FluidResVolProd
-            tmp_values[6] = st.get("FOIT", 0.0); // WellCumDataType::OilInj
-            tmp_values[7] = st.get("FWIT", 0.0); // WellCumDataType::WaterInj
-            tmp_values[8] = st.get("FGIT", 0.0); // WellCumDataType::GasInj
-            tmp_values[9] = st.get("FVIT", 0.0); // WellCumDataType::FluidResVolInj
-        } else {
-            tmp_values[2] = get("GOPT"); // WellCumDataType::OilProd
-            tmp_values[3] = get("GWPT"); // WellCumDataType::WaterProd
-            tmp_values[4] = get("GGPT"); // WellCumDataType::GasProd
-            tmp_values[5] = get("GVPT"); // WellCumDataType::FluidResVolProd
-            tmp_values[6] = get("GOIT"); // WellCumDataType::OilInj
-            tmp_values[7] = get("GWIT"); // WellCumDataType::WaterInj
-            tmp_values[8] = get("GGIT"); // WellCumDataType::GasInj
-            tmp_values[9] = get("GVIT"); // WellCumDataType::FluidResVolInj
+    for (const auto& gname : this->schedule_.groupNames()) {
+        std::vector<Scalar> values(WellCumDataType::numWCValues, 0.0);
+        std::vector<std::string> names(WellCumDataType::numWCNames, "");
```

Comment:
better use default ctor

Reply:
> better use default ctor

Agreed.  I've pushed an update to that effect.